integration-tests/api/__tests__/admin/product.js

```diff
@@ -32,6 +32,50 @@ let {
 
 jest.setTimeout(50000)
 
+const getProductFixture = () => ({
```

Comment:
question: The reason I moved this into the `medusaIntegrationTestRunner` a few PRs ago was that the `breaking` helper only works outside of the runner, if you use `process.env`. Passing the V2 FF in `env` will not work as expected (I am quite sure).

Reply:
Ah sorry, I missed this part!